students/M_Sunday/lesson09/Mailroom_Part_5.py

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/python
+from operator import itemgetter
+
+# Lesson 09, Mailroom Part 5 Assignment
+
+
+class UserInputs(object):
+
+    @staticmethod
+    def get_menu_option():
+        sel_ = input("------> "
+                     "Select an option from the menu: ")
+        return sel_
+
+    @staticmethod
+    def get_donation_amount(donor_name_):
+        amount_ = input("\nEnter the donation amount for {} ".format(donor_name_))
+        return amount_
+
+    @staticmethod
+    def get_donors_name():
+        name_ = input("\nPlease enter the donor's full name: ")
+        return name_
+
+
+class DonorData(object):
+
+    def __init__(self):
+        self.donor_data_ = {}
+
+    @property
+    def donor(self):
+        pass
+
+    @donor.setter
+    def donor(self, name):
+        self.donor_data_[name] = []
+
+    def delete_donor(self, name):
+        self.donor_data_.pop(name)
+
+    def add_donation(self, name, donation):
+        if not(name in self.donor_data_):
+            self.donor_data_[name] = donation
+        else:
+            self.donor_data_[name].append(donation)
+            print("adding printer")
+
+    def list_display(self):
+        print("\n---- List of all Donors ----")
+        [print("- {}".format(item)) for item in self.donor_data_]
```

Comment:
it is preferred to use regular for loop if you are not consuming the return value